tests/unicode.php

```diff
@@ -0,0 +1,39 @@
+--TEST
+Test V8::executeString() : Check if imported code works with umlauts
+--SKIPIF--
+<?php require_once(dirname(__FILE__) . '/skipif.inc'); ?>
+--FILE--
+<?php
+
+# maybe more characters (e.g. from http://www.ltg.ed.ac.uk/~richard/unicode-sample.html?)
+$unicode = 'äöüßÜÄÖÜß€áàâÁÀÂµ²³▁▂▃▄▅▆▇█    ㌀ ㌁ ㌂ ㌃';
+
+# insert unicode via snapshot
+$snapshot = V8Js::createSnapshot("var snapshot = {unicode: '" . $unicode . "'}");
```

Comment:
Keep in mind that V8Js can be compiled with snapshot support turned off.

You might want to create a second test that focuses on testing unicode support in combination with snapshots (and just skip it if V8 doesn't have support for snapshots)